Source/SVONavigation/Private/PathFinding/SVOPathFinder.cpp

```diff
@@ -72,8 +72,11 @@ ENavigationQueryResult::Type FSVOPathFinder::GetPath( FSVONavigationPath & navig
 
         if ( const auto * path_finder = GetPathFindingAlgorithm( navigation_query_filter_copy ) )
         {
-            const FSVOPathFindingParameters params( *volume_navigation_data, start_location, end_location, *navigation_query_filter_copy );
-            return path_finder->GetPath( navigation_path, params );
+            const auto params = FSVOPathFindingParameters::Initialize( *volume_navigation_data, start_location, end_location, *nav_query_filter );

```

Comment:
should we not use navigation_query_filter_copy instead of nav_query_filter?

Reply:
Done